### ISSUE TITLE
https://www.ovirt.org/documentation/how-to/networking/bonding-vlan-bridge/

### ISSUE THREAD

**Issue by hdatma** [oVirt/ovirt-site#1802]
### Description of the problem

1. Missing date of last update. 
2. Device names (em1, em2, ...) suggest the document needs revision. 
3. Are the instructions correct wrt current distributions?

**Comment by hdatma** [comment]
The instructions lead to a messy configuration. 
I attach two scripts, one to test the instructions, the other to undo it all. 

[nm-on.txt](https://github.com/oVirt/ovirt-site/files/2577601/nm-on.txt)
[nm-off.txt](https://github.com/oVirt/ovirt-site/files/2577602/nm-off.txt)

**Comment by gregsheremeta** [comment]
This is outdated and shouldn't be used. I added a warning to the page to indicate that:

```
This developer documentation is outdated, but provides historical context. 

It is not user documentation and should not be treated as such. 

Documentation is available here.
```

Fixed in #1840
